manifests/0000_26_cloud-controller-manager-operator_03_rbac_provider.yaml

```diff
@@ -8,57 +8,6 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
 
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: cloud-controller-manager
-  namespace: kube-system
-  annotations:
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
-rules:
-  - apiGroups:
-      - ""
-    resources:
-      - serviceaccounts
-    resourceNames:
-      - node-controller
-      - service-controller
-    verbs:
-      - get
-
-  # This grants CCMs permission to list and read all secrets in kube-system.
-  # This is required by client builder to fetch the ServiceAccount token
-  # secret. Ideally we would restrict this to the token secrets of specific
-  # ServiceAccounts.
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-    verbs:
-      - list
-      - get
-      - watch
-
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: cloud-controller-manager
-  namespace: kube-system
-  annotations:
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
-roleRef:
-  kind: Role
-  name: cloud-controller-manager
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-  - kind: ServiceAccount
-    namespace: openshift-cloud-controller-manager
-    name: cloud-controller-manager
-
```

Comment:
AWS covers these RBAC changes with upaded cluster role [here](https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/15/files#diff-0856a88d16f3aa369e7f5d7fc7d1e356008fc796a5800ad4da0e304113f6af2cR60). I consider removing this for now, as for example Azure and vSphere both are using same clusterRole, which will work for OpenStack too. We will investigate and possible reduce the scope of it later.

Reply:
Ok, sure. As I mentioned elsewhere I think we could potentially eliminate these entirely later.